ERCS/erc-7007.md

```diff
@@ -47,33 +52,34 @@ interface IERC7007 is IERC165, IERC721 {
         bytes indexed prompt,
         bytes indexed aigcData,
         string uri,
-        bytes proof
+        bytes proofOrState
     );
 
     /**
-     * @dev Mint token at `tokenId` given `prompt`, `aigcData`, `uri` and `proof`.
+     * @dev Mint token at `tokenId` given `prompt`, `aigcData`, `uri` and `proofOrState`.
+     * @dev `proofOrState` means that we input the zk proof when using zkML and the final inference state when using opML as the verification method.
      *
      * Requirements:
      * - `tokenId` must not exist.'
      * - verify(`prompt`, `aigcData`, `proof`) must return true.
      *
      * Optional:
-     * - `proof` should not include `aigcData` to save gas.
+     * - `proofOrState` should not include `aigcData` to save gas.
      */
     function mint(
         bytes calldata prompt,
         bytes calldata aigcData,
         string calldata uri,
-        bytes calldata proof
+        bytes calldata proofOrState
     ) external returns (uint256 tokenId);
 
     /**
-     * @dev Verify the `prompt`, `aigcData` and `proof`.
+     * @dev Verify the `prompt`, `aigcData` and `proofOrState`.
      */
     function verify(
         bytes calldata prompt,
         bytes calldata aigcData,
-        bytes calldata proof
+        bytes calldata proofOrState
```

Comment:
this is a good way to integrate. just a question:
I assume the 'state' means the state root hash in opml, provided by the submitter, but then can we still call it 'verify'? would like to see how does the opml challenger interact with this interface.